client/src/components/pages/Posts/style.css

```diff
@@ -0,0 +1,68 @@
+.posts {
+  padding-top: 80px;
+  width: 85%;
+  margin: 0 auto;
+  height: calc(100vh - (115px + 70px));
+}
+
+#root {
```

Comment:
@IsraaSulaiman If this (the min-height for #root) is a global style, it might be better to move this into App.css so it's easier to find later.

Reply:
Thanks. I'll move the root style to App.css